package.json

```diff
@@ -33,11 +33,14 @@
     }
   },
   "scripts": {
-    "preinstall": "git submodule update --init --recursive && git submodule foreach --recursive git pull origin master && yarn --cwd ccd-definition-processor install",
+    "setup": "git submodule update --init --recursive && git submodule foreach --recursive git pull origin master && yarn --cwd ccd-definition-processor install",
```

Comment:
Had to move this because the docker container doesn't have `git` and it was trying to run this command because it's part of `yarn install`